gui/qt/main_window.py

```diff
@@ -2630,19 +2636,20 @@ def on_nz():
         nz.valueChanged.connect(on_nz)
         gui_widgets.append((nz_label, nz))
 
-        def on_maxfee(x):
-            m = maxfee_e.get_amount()
-            if m: self.config.set_key('max_fee_rate', m)
-            self.fee_slider.update()
-        def update_maxfee():
-            maxfee_e.setDisabled(False)
-            maxfee_label.setDisabled(False)
-        maxfee_label = HelpLabel(_('Max static fee'), _('Max value of the static fee slider'))
-        maxfee_e = BTCkBEdit(self.get_decimal_point)
-        maxfee_e.setAmount(self.config.max_fee_rate())
-        maxfee_e.textChanged.connect(on_maxfee)
-        update_maxfee()
-        fee_widgets.append((maxfee_label, maxfee_e))
+        def on_customfee(x):
+            m = customfee_e.get_amount()
+            if (m is None):
+                self.fee_slider.update()
```

Comment:
The pattern of update or deactivate should perhaps be encapsulated in BTCkEdit2, and not done in this file.

Reply:
That doesn't makes sense to me.  BTCkEdit2 is a method in the amountedit gui , used in preferences.  Fee slider is in a totally different place, on the transaction dialouge.